main.nf

```diff
@@ -564,7 +564,7 @@ process get_software_versions {
     freebayes --version &> v_freebayes.txt 2>&1 || true
     freec &> v_controlfreec.txt 2>&1 || true
     gatk ApplyBQSR --help &> v_gatk.txt 2>&1 || true
-    msisensor &> v_msisensor.txt 2>&1 || true
+    msisensor-pro &> v_msisensor.txt 2>&1 || true
```

Comment:
let's keep `msisensor` here, `msisensor-pro` is not in the main container, but just used for the msi processes

Reply:
You forgot about this one ;-)

Reply:
Thought you wanted to keep `msisensor' there instead of `msisensor-pro'